src/python/pants/backend/python/tasks/pytest/plugin.py

```diff
@@ -8,6 +8,7 @@
 
 import json
 import os
+from zlib import crc32
```

Comment:
Nit: In this instance, think it’s more readable to import `zlib` so that it’s used as `zlib.crc32`.

Reply:
Algorithms like crc32 and sha1 are very well defined, and are probably a good counterexample to the need for absolute imports.